Sum_of_numbers_string.py

```diff
@@ -0,0 +1,22 @@
+// ADDING ALL NUMBERS IN A STRING.
+
+CODE:
+
+st = input("Enter a string: ")
+a = "" 
+Sum = 0
```

Comment:
all variable names should be lowercase separated by underscores. Since sum is a reserved keyword for a builtin function, use something like total instead.